File: AR-VR/saumyasinha31_SaumyaSinha_2125cse1144_2/GettingStarted/Print Fibonacci Numbers Till N.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
      int a=0;
    int b=1;
    int c= a+b;
    int n;
    cin >> n;
      
     cout<<0<<endl<<1<<endl;
        for(int i=2;i<n;++i){
            c=a+b;
            cout<<c<<endl;
            a=b;
            b=c;
        }
    
  return 0;  //write your code here
    
}

File: Web/EvaMittal_EvaMittal_2125cs1187_2/GettingStarted/PrintFibonacciNumbersTillN.java
import java.util.*;
public class PrintFibonacciNumbersTillN {
    public static void main(String[] args) {
        // write your code here
        Scanner scn= new Scanner(System.in);
        int n = scn.nextInt();
        scn.close();
        int i, a=0, b=1,c;
        System.out.println(a);
        System.out.println(b);
        for(i=3;i<=n;i++)
        {
            c=a+b;
            System.out.println(c);
            a=b;
            b=c;
        }
     }
    
}
